spec/luassert_helper.lua

```diff
@@ -1,7 +1,20 @@
 local busted = require('busted')
 local assert = require('luassert.assert')
+local spy = require('luassert.spy')
+local stub = require('luassert.stub')
 local util = require ('luassert.util')
 local say = require('say')
+local tablex = require('pl.tablex')
+local deepcopy = tablex.deepcopy
+
+local original_stub_new = deepcopy(stub.new)
+
+stub.new = function(object, key, ...)
+  if object and spy.is_spy(object[key]) then
+    object[key]:revert()
```

Comment:
I think we just could throw an error and let the developer handle it. I think that is more valuable than doing something magical in the background.